utils/api/benefits/oasBenefit.ts

```diff
@@ -578,7 +578,9 @@ export class OasBenefit extends BaseBenefit<EntitlementResultOas> {
     ) {
       // your Deferral Options
 
-      text += `<p class='mb-2 mt-6 font-bold text-[24px]'>${this.translations.detail.yourDeferralOptions}</p>`
+      if (this.deferral) {
```

Comment:
this breaks all the other scenarios

Reply:
@MarcoGoC I moved the heading into the respective if statements. I am unable to see of a way that we can append the heading outside of the two if statements without breaking other scenarios. Open to suggestions.